docs/design/live_migration.md

```diff
@@ -0,0 +1,176 @@
+# KubeVirt virtual machine bridge binding pod network seamless live migration
+
+
+## Introduction
+
+At kubevirt every VM is executed inside a "virt-launcher" pod; how the "virt-launcher" pod net interface is "bound" to the VM is specified by the network binding at the VM spec, ovn-kubernetes support live migration of the KubeVirt network bridge binding. 
+
+At KubeVirt bridge binding the virt-launcher interface is replaced by a bridge with one leg on the pod interface and the other at the VM tap device. When IPAM is configured on the pod interface, a DHCP server is started to advertise the pod's IP to DHCP aware VMs.
+
+Benefict of the bridge binding is that is able to expose the pod IP to the VM as expected by most users also pod network bridge binding live migration is implemented by ovn-kubernetes by doing the following:
+- Do IP assignemnt with ovn-k but skip the CNI part that configure it at pod's netns veth.
+- Do not expect the IP to be on the pod netns.
+- Add the ability at ovn-k controllers to migrate pod's IP from node to node.
+
+
+## Implementation 
+
+To implement live migration ovn-kubernetes do the following:
+- Deliver IP address at VMs configuring logical switch port DHCP options, the kubevirt DHCP server is deactivated.
```

Comment:
send DHCP replies advertising the allocated IP address to the guest VM (via OVN-Kubernetes DHCP options configured for the logical switch ports).
```suggestion
- Deliver IP address at VMs configuring logical switch port DHCP options, the kubevirt DHCP server is deactivated.
```

... or something like that.

The kubevirt DHCP part, while interesting, has nothing to do with OVN.Kubernetes, right ?

Reply:
> The kubevirt DHCP part, while interesting, has nothing to do with OVN.Kubernetes, right ?

Right, is just to understand that also KubeVirt is using DHCP too to offer the ip address, so reader see that this feature copy how is done at KubeVirt.